src/driver/test/CrudActor.yml

```diff
@@ -0,0 +1,24 @@
+SchemaVersion: 2018-07-01
+  
+Actors:
+- Name: CrudActor
+  Type: CrudActor
+  Database: mydb
+  ExecutionStrategy:
+    ThrowOnFailure: true
+  Phases:
+  - Repeat: 1
+    Collection: test
+    Operations:
+    - OperationName: bulkWrite
+      OperationCommand:
+        WriteOperations:
+        - WriteCommand: insertOne
+          Document: { a: 1 }
+        - WriteCommand: updateOne
+          Filter: { a: 1 }
+          Update: { $set: { a: 5 } }
+          Options:
+            WriteConcern:
+              Level: majority
+              TimeoutMillis: 5000
```

Comment:
plz extend this example to include the opeartions that are supported and show a variety of value-generator usages as well.